packages/langium/src/utils/regexp-utils.ts

```diff
@@ -140,7 +140,7 @@ export function isMultilineComment(regexp: RegExp | string): boolean {
 
 export function isWhitespace(value: RegExp | string): boolean {
     const regexp = typeof value === 'string' ? new RegExp(value) : value;
-    return regexp.test(' ');
+    return [' ', '\n', '\t'].some((ws) => regexp.test(ws));
```

Comment:
Suggestion: There are a few more (but more rarely used) line feed characters that should probably be included in there
```suggestion
    return [' ', '\n', '\t', '\v', '\r'].some((ws) => regexp.test(ws));
```

Reply:
I considered adding them but thought they are most likely not used by anyone. I am not against including those, but then again `\s` also accepts other characters, such as form feed (`\f`), non-breaking space (`U+00A0`), and a few others. Based on what do we decide which to include here 😅?

> For reference, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions/Character_classes) specifies that `\s` is equivalent to `[\f\n\r\t\v\u0020\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]`

Reply:
It's unlikely that people use them, but I think it still makes sense to include all of them. Using the `split('')` approach sounds reasonable. Can you write a comment explaining that that this is supposed to be `\s` with a link to MDN?

Reply:
Sure thing